keyboards/massdrop/ctrl/keymaps/alexschneider/config_led.c

```diff
@@ -0,0 +1,82 @@
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
I don't think you need this file at all. It's identical to the one in the keyboard folder.